libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.h

```diff
@@ -337,6 +308,24 @@ FOUNDATION_EXTERN NSString * const kSFLoginHostChangedNotificationUpdatedHostKey
  will try to determine that.
  */
 - (void)userChanged:(SFUserAccountChange)change;
+/**
+ *
+ * @return the default Credentials
+ */
+- (SFOAuthCredentials *)oauthCredentials;
+
+/** Invoke this method to replace the SFDefaultUserAccountPersister with a custom implementation.
+ *
+ * @param persister An object that implements SFUserAccountPersister
+ */
++ (void)setAccountPersisterClass:(nullable Class) persister;
```

Comment:
Maybe it would be more straightforward to make this a property? Or maybe an initializer parameter?